Resources/Prototypes/Recipes/Lathes/electronics.yml

```diff
@@ -983,3 +983,13 @@
   materials:
     Steel: 100
     Glass: 500
+
+- type: latheRecipe
+  id: SalvageMagnetMachineCircuitboard
```

Comment:
no reason for this to take extra steel or be 10 seconds, just inherit the gold electronics recipe